stream-chat-android-ui-components/src/main/kotlin/com/getstream/sdk/chat/viewmodel/messages/MessageListItemLiveData.kt

```diff
@@ -217,10 +246,14 @@ internal class MessageListItemLiveData(
      * @return Full list of [MessageListItem] to represent the state.
      */
     private fun buildItemsList(): List<MessageListItem> {
-        return getLoadingMoreItems() + messageItemsWithReads + typingItems
+        return if (loadingMoreOlderItems) {
```

Comment:
I haven't seen loading indicators in the message list for quite a while. Are they handled in the UI?

Reply:
it seems we do not have a ViewHolder for the loading spinners.

Reply:
Created a ticket for that:
https://github.com/GetStream/stream-chat-android/issues/4241